Sources/Mantis/Protocols/TransformDelegate.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  TransformDelegate.swift
+//  Mantis
+//
+//  Created by Richard Shane on 21/3/2024.
+//
+
+import Foundation
+
+protocol TransformDelegate: AnyObject {
+    func undoManager() -> UndoManager
```

Comment:
I suggest to rename it to `getUndoManager()`

Reply:
changes made